src/components/Alert.js

```diff
@@ -10,10 +10,10 @@ export const Alert = (type) => {
       });
       break
     case 'signupError':
-      toast('Could not signup', {
+      toast('Could not signup, Username must be greater than or equal to 5 characters', {
         autoClose: 3000,
         className: "alert alert-rose",
-        render: '<h2><strong>Could not login</strong></h2>'
+        render: '<h2><strong>Could not signup, Username must be greater than or equal to 5 characters</strong></h2>'
```

Comment:
as these are the same message repeated twice, I would have made them a variable and just used it in both places. Keep it DRY.

Reply:
@Jeffchiucp you aren't checking the reason there is an error?  This toast could be sent for any error.  That's why I was suggesting that you control for this in the input validation rules and disable the Submit button until the validation rules have been met for each input.